include/openssl/x509v3.h

```diff
@@ -375,8 +374,12 @@ struct ISSUING_DIST_POINT_st {
 
 # define EXFLAG_INVALID_POLICY   0x800
 # define EXFLAG_FRESHEST         0x1000
-/* Self signed */
-# define EXFLAG_SS               0x2000
+# define EXFLAG_SS               0x2000 /* cert is apparently self-signed */
+
+# define EXFLAG_BCONS_CRITICAL   0x10000
+# define EXFLAG_AKID_CRITICAL    0x20000
+# define EXFLAG_SKID_CRITICAL    0x40000
+# define EXFLAG_SAN_CRITICAL     0x80000
```

Comment:
Is it useful to put this in a public header? As far as I know, you can't get to those flags anymore since 1.1.0.

Reply:
But putting them in an other header will probably lead to confusion.

Reply:
Well, I put them there simply because also all other `#define EXFLAG_*` have been placed there, too.
Most of them are documented and may be of good external use, 
while others, including these four but also EXFLAG_NSCERT and EXFLAG_SET, 
but also the documented flags EXFLAG_CRITICAL and EXFLAG_INVALID_POLICY, are likely only used internally.
This could be cleaned up in a later PR.